ansibleflow/app.py

```diff
@@ -50,13 +50,29 @@ def setup_argument_parser():
     )
 
     run_parser = subparsers.add_parser('run')
-    run_parser.add_argument('--env', type=str, default='default')
     run_parser.add_argument(
         'run_action',
-        nargs='*',
+        type=str,
         default=True,
         help=argparse.SUPPRESS
     )
+    run_parser.add_argument(
+            '--env',
+            type=str,
+            default='default')
+    run_parser.add_argument(
+            "-v",
+            "--verbosity",
+            action="count",
+            default=0,
+            help="increase output verbosity")
+    run_parser.add_argument(
+            '--file',
+            type=str,
+            default=None)
+    run_parser.add_argument(
+            '--extra-vars',
+            nargs='*')
```

Comment:
Why are extra-vars being passed in via the cli? Shouldn't this just be a target or environment option? The whole purpose of ansible-flow is to pre-configure everything so people can run a target against an environment the same way everytime.